### PR TITLE
typos

### PR DESCRIPTION
**Pull request by noname007**
I hereby granted the copyright of the changes in this pull request
to the authors of this lua-nginx-module project.

### CODE REVIEW

**Comment by detailyang** [comment]
@noname007 You should edit the file [doc/HttpLuaModule.wiki](https://github.com/openresty/lua-nginx-module/blob/master/doc/HttpLuaModule.wiki) instead. The `README.markdown` is generated automatically by `wiki2markdown.pl doc/HttpLuaModule.wiki`.

File: README.markdown [hunk 1]
```diff
@@ -3565,7 +3565,7 @@ When being used in the context of [init_worker_by_lua*](#init_worker_by_lua), th
 
 The `ngx.ctx` lookup requires relatively expensive metamethod calls and it is much slower than explicitly passing per-request data along by your own function arguments. So do not abuse this API for saving your own function arguments because it usually has quite some performance impact.
 
-Because of the metamethod magic, never "local" the `ngx.ctx` table outside your Lua function scope on the Lua module level level due to [worker-level data sharing](#data-sharing-within-an-nginx-worker). For example, the following is bad:
+Because of the metamethod magic, never "local" the `ngx.ctx` table outside your Lua function scope on the Lua module level  due to [worker-level data sharing](#data-sharing-within-an-nginx-worker). For example, the following is bad:
```
Comment (detailyang) [inline README.markdown]: duplicated space?

Comment (noname007) [inline README.markdown]: yes

**Comment by agentzh** [comment]
@noname007 Merged with fixes for issues pointed out by @detailyang. Thanks!

**Pull request closed without merging**